lighthouse-core/report/html/report-styles.css

```diff
@@ -548,11 +551,66 @@
 }
 
 .lh-metric__description {
+  display: none;
   color: var(--secondary-text-color);
+  padding: var(--expandable-padding);
+}
+.lh-audit-group--metrics__show-descriptions .lh-metric__description {
+  display: block;
 }
 
 .lh-metric__value {
   white-space: nowrap; /* No wrapping between metric value and the icon */
+  font-weight: 500;
+}
+
+.lh-metrics-toggle {
+  position: relative;
+}
+.lh-metrics-toggle input {
+  cursor: pointer;
+  left: 0;
+  opacity: 0;
+  position: absolute;
+  top: 0;
+  width: 74px;
+  height: 28px;
+}
+.lh-metrics-toggle label {
+  display: flex;
+  background-color: #eee;
+  border-radius: 20px;
+  overflow: hidden;
+}
+.lh-metrics-toggle label div {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 2px 5px;
+}
+.lh-metrics-toggle input:focus + label {
+  outline: -webkit-focus-ring-color auto 3px;
+}
+.lh-metrics-toggle input:not(:checked) + label .lh-metrics-toggle-less,
+.lh-metrics-toggle input:checked + label .lh-metrics-toggle-more {
+  background-color: var(--color-blue);
+}
+.lh-metrics-toggle-lines {
+  fill: #7F7F7F;
+}
+.lh-metrics-toggle input:not(:checked) + label .lh-metrics-toggle-less .lh-metrics-toggle-lines,
+.lh-metrics-toggle input:checked + label .lh-metrics-toggle-more .lh-metrics-toggle-lines {
```

Comment:
instead of this, you could override a css var up in the conditional L595 rule ... 

```css
  --color-lines: var(--color-white);
```